README.md

```diff
@@ -10,4 +10,4 @@ a folder within the application directory.
 * Go into the SCATT directory with the command `cd cs-5666-scatt-gp`.
 * Enter `make scatt` and hit return.
 * Enter the name of your SCATT submissions folder when prompted.
-* View the report output.
+* To view the report, open the file named `Report.txt`.
```

Comment:
@KaraBeason Do you think we should mention where the file is outputted to? And now that I think about it for the next iteration we might want to make the screen say the report is done so go look at it since everything auto runs the user might be waiting for it to work after it's already done.